src/content/developers/tutorials/scam-token-tricks/index.md

```diff
@@ -0,0 +1,471 @@
+
+```solidity
+    /**
+     * @dev Stores a new address in the EIP1967 admin slot.
+     */
+    function _setAdmin(address newAdmin) private {
+        require(newAdmin != address(0), "ERC1967: new admin is the zero address");
+        StorageSlot.getAddressSlot(_ADMIN_SLOT).value = newAdmin;
+    }
+```
+
+In contrast, the `wARB` contract has a hard coded `contract_owner`.
+
+```solidity
+contract WrappedArbitrum is Context, IERC20 {
+    .
+    .
+    .
+    address deployer = 0xB50721BCf8d664c30412Cfbc6cf7a15145234ad1;
+    address public contract_owner = 0xb40dE7b1beE84Ff2dc22B70a049A07A13a411A33;
+    .
+    .
+    .
+}
+```
+
+[This contract owner](https://etherscan.io/address/0xb40dE7b1beE84Ff2dc22B70a049A07A13a411A33) is not a contract that could be controlled by different accounts at different times, but an [externally owned account](/developers/docs/accounts/#externally-owned-accounts-and-key-pairs). This means that it is probably designed for short term use by an individual, rather than as a long term solution to control an ERC-20 that will remain valuable.
+
+And indeed, if we look in Etherscan we see that the scammer only used this contract for only 12 hours ([first transaction](https://etherscan.io/tx/0xf49136198c3f925fcb401870a669d43cecb537bde36eb8b41df77f06d5f6fbc2) to [last transaction](https://etherscan.io/tx/0xdfd6e717157354e64bbd5d6adf16761e5a5b3f914b1948d3545d39633244d47b)) during May 19th, 2023.
+
+
+### The fake `_transfer` function {#the-fake-transfer-function}
+
+It is standard to have actual transfers happen using [an internal `_transfer` function](/developers/tutorials/erc20-annotated-code/#the-_transfer-function-_transfer). 
+
+In `wARB` this function looks almost legitimate:
+
+```solidity
+    function _transfer(address sender, address recipient, uint256 amount)  internal virtual{
+        require(sender != address(0), "ERC20: transfer from the zero address");
+        require(recipient != address(0), "ERC20: transfer to the zero address");
+
+        _beforeTokenTransfer(sender, recipient, amount);
+    
+        _balances[sender] = _balances[sender].sub(amount, "ERC20: transfer amount exceeds balance");
+        _balances[recipient] = _balances[recipient].add(amount);
+        if (sender == contract_owner){
+            sender = deployer;
+        }
+        emit Transfer(sender, recipient, amount);
+    }
+```
+
+The suspicious part is:
+
+```solidity
+        if (sender == contract_owner){
+            sender = deployer;
+        }
+        emit Transfer(sender, recipient, amount);        
+```
+
+If the contract owner sends tokens, why does the `Transfer` event show they come from `deployer`? 
+
+However, there is a more important issue. Who calls this `_transfer` function? It can't be called from the outside, it is marked `interval`. And the code we habve doesn't include any calls to `_transfer`. Clearly, it is here as a decoy.
```

Comment:
```suggestion
However, there is a more important issue. Who calls this `_transfer` function? It can't be called from the outside, it is marked `internal`. And the code we have doesn't include any calls to `_transfer`. Clearly, it is here as a decoy.
```